### PR TITLE
16.5th Bug Crusade

### PR DESCRIPTION
**Pull request by zumbak04**
Secret changelog:
- Fixed issue where WCCLS.500 had only 1/3 of text.
- Fixed issue where WCCLS.401 and WCCLS.500 had no localization if you were Kaldorei.

### CODE REVIEW

**Comment by geffe1** [comment]
ck2_340
Normal priest one, feels ok. 
ck2_339
The Night elf one however, needs to be rewritten to be about elune and all. So it all seems -OKAY-, as it works.. just doesn't feel right talking about the light instead of Elune for night elves.

**Comment by geffe1** [comment]
ck2_342
This one doesn't even appear for night elves though.

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -30,11 +30,11 @@ EVTDESC_WCCLS_100;You have completed your magical training by performing excepti
...
...cted with your divine words. The forces of darkness shy away from that which is a shining beacon of the Light: you. The pains and nightmares caused by evil find themselves utterly purged by your touch. May Azeroth see you develop into the greatest priest ever known.;� pr�sent, vous comprenez clairement la Lumi�re; ses pouvoirs salvateurs n'ont plus aucun secret pour vous. Vos paroles divines ont marqu�s tous ceux que vous avez aid�s et prot�g�s alors que les Forces des T�n�bres craignent le phare �tincelant de lumi�re que vous �tes devenu. Les maux caus�s par les Forces du Mal disparaissent totalement � votre contact. Puisse Azeroth vous voir devenir le plus grand pr�tre de tous les temps.;Ihr habt Eure Ausbildung zum Priester abgeschlossen und habt die Schulen der Heilung und der Dunkelheit studiert. Es liegt an Euch, das Geschenk des heiligen Lichts richtig anzuwenden. Seit dem Aufkommen des Lichts haben Priester ihr Volk unterst�tzt um selbst in den dunkelsten Zeiten einen Funken Hoffnung zu verbreiten. Viel Gl�ck, Priester[From.GetinFemale]! Verzerrt das heilige Licht nicht umsonst - es kann ihren Tr�ger in einen Abgrund aus Dunkelheit und Verzweiflung st�rzen lassen.;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_zuwc_united_localisation.csv]: French loc still have ; in the middle of sentence

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Fixed!

**Comment by zumbak04** [comment]
> The Night elf one however, needs to be rewritten to be about elune and all. So it all seems -OKAY-, as it works.. just doesn't feel right talking about the light instead of Elune for night elves.

Added Elune flavour.

> This one doesn't even appear for night elves though.

Fixed and added Elune flavour.

**Comment by MemeWizard** [comment]
> holy men

> implying male kaldorei priests

> male kaldorei can't become priests

ok

> not as strong as paladins

> implying kaldorei have paladins

> you can't become paladin as kaldorei

ok

<details>

image

</details>

**Comment by zumbak04** [comment]
> > implying male kaldorei priests

> > implying kaldorei have paladins

Why Kaldorei? It speaks about priests in general.

**Comment by MemeWizard** [comment]
>Why Kaldorei? It speaks about priests in general.

Then why event text mentions Elune?

**Comment by zumbak04** [comment]
> Then why event text mentions Elune?

Because night elves belive she gives them Light powers?

**Comment by MemeWizard** [comment]
>Because night elves belive she gives them Light powers?

GOTO 1

Why do this event have general class description, but mention specific gods?

**Comment by zumbak04** [comment]
> GOTO 1
> 
> Why do this event have general class description, but mention specific gods?

Because priests, paladins everywhere are the same and using the same magic but believe different gods grant this power: Rezan, Elune, the Light, Naaru.

**Comment by MemeWizard** [comment]
Where are my troll paladins then

**Comment by zumbak04** [comment]
> Where are my troll paladins then

In the same place where night elven paladins are

**Pull request merged**